### ISSUE TITLE
Discussion with the AngularUI guys

### ISSUE THREAD

**Issue by ProLoser** [jimrhoskins/ngmodules.org#5]
Hey @jimrhoskins 

So I don't remember exactly if you were the guy I was talking to when you were developing this project on IRC as I was discussing with someone using bower and creating an Angular-based npm system.

Anyway, we at AngularUI are trying to figure out how we may end up changing our project structure and thought you might be interested or have some input as it may involve you and ngmodules.org (if you're interested).

Let me know if you'd be up to chat, we will probably try to organize a hangout.

**Comment by jimrhoskins** [comment]
Hi @ProLoser 

I was most likely that guy. I'd love to chat with you about modules and structuring. My g+ http://gplus.to/jimrhoskins

Thanks!

**Comment by ProLoser** [comment]
Go ahead and hop over to this thread: https://groups.google.com/forum/?fromgroups#!category-topic/angular-ui/ui/4tEbsfqosDc

### TL;DR:

AngularUI needs to reorganize and become more modular but inter-dependent. This may mean breaking each module into it's own project/repo and simply adding tags. 1 build / dependency system across all projects.

Rather than purely focusing on our own build systems that bundle tagged modules together, people may want to hook into our build or dependency management system.

You once spoke about building an angular-themed cli to sit on top of bower. If we focused on this solution we could drop concerns for our own build system and packager.

If you were interested, we'd add you to the team and ask you to champion the cli build / package system that would run alongside ngmodules.org. Of course you would have the added benefit of the AngularUI team members to help too.

**Comment by ProLoser** [comment]
I realize this may sound extremely forward so feel free to turn it down, but I went ahead and added you to the AngularUI and created a new 'directory' team that will be focused on the builder and related discussions.

If you are interested in 'joining forces' (lol) you can transfer ngmodules to angular-ui and I will add it to the directory team. You would of course retain full control over the project and all major decisions and would not be asked to share any information regarding the production server.
If however you feel you want to keep the primary fork under your own namespace I will fully understand :)

**Comment by jimrhoskins** [comment]
Sorry about the delayed. I got slammed with work and personal stuff this last week and my OS time has suffered.

I think this could work. There's a parallel discussion happening on the ui mailing list, so I'll give more info there. I'll close this ticket, and we can discuss further either on the angular ui mailing list, or via github on angular-ui projects.

Thanks, looking forward to getting back into this.
